packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -565,6 +569,26 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
     );
   };
 
+  /* @conditional-compile-remove(breakout-rooms) */
+  const assignedBreakoutRoom = useSelector(getAssignedBreakoutRoom);
+  /* @conditional-compile-remove(breakout-rooms) */
+  const breakoutRoomSettings = useSelector(getBreakoutRoomSettings);
+
+  let latestNotifications = props.latestNotifications;
+  /* @conditional-compile-remove(breakout-rooms) */
+  // Filter out breakout room notification that prompts user to join breakout room when in mobile view. We will
+  // replace it with a non-dismissible banner
+  latestNotifications = props.mobileView
+    ? (latestNotifications ?? []).filter((n) => n.type !== 'assignedBreakoutRoomOpenedPromptJoin')
+    : latestNotifications;
+
+  /* @conditional-compile-remove(breakout-rooms) */
+  const movingToBreakoutRoomAutomatically =
+    assignedBreakoutRoom?.autoMoveParticipantToBreakoutRoom &&
+    latestNotifications?.find(
+      (n) => n.type === 'assignedBreakoutRoomOpened' || n.type === 'assignedBreakoutRoomChanged'
```

Comment:
Can we call this `notification` instead of `n`? just easier to read in the future

Reply:
Yup. Updated